app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+    first_file, second_file = command_list[1], command_list[2]
+
+    if first_file != second_file:
+        with (open(first_file, "r") as file_from,
```

Comment:
You can simplify this by using commands from built in `shutil` library.
Also how does your code handle invalid input (non-existing files, invalid file names)?

Reply:
> You can simplify this by using commands from built in `shutil` library. Also how does your code handle invalid input (non-existing files, invalid file names)?

I'll checkout shutil lib after the morning lesson, and I add try/except to handle invalid inputs if shutil doesn't do it also.